client/web/src/charts/views/categorical/CategoricalChart.story.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+
+import { Meta } from '@storybook/react'
+
+import { WebStory } from '../../../components/WebStory'
+import { CategoricalBasedChartTypes } from '../../types'
+
+import { CategoricalChart } from './CategoricalChart'
+
+export default {
+    title: 'web/charts/CategoricalChart',
+    decorators: [story => <WebStory>{() => story()}</WebStory>],
+} as Meta
+
+interface LanguageUsageDatum {
+    name: string
+    value: number
+    fill: string
+    linkURL: string
+}
+
+const LANGUAGE_USAGE_DATA: LanguageUsageDatum[] = [
+    {
+        name: 'JavaScript',
+        value: 422,
+        fill: '#f1e05a',
+        linkURL: 'https://en.wikipedia.org/wiki/JavaScript',
+    },
+    {
+        name: 'CSS',
+        value: 273,
+        fill: '#563d7c',
+        linkURL: 'https://en.wikipedia.org/wiki/CSS',
+    },
+    {
+        name: 'HTML',
+        value: 129,
+        fill: '#e34c26',
+        linkURL: 'https://en.wikipedia.org/wiki/HTML',
+    },
+    {
+        name: 'Markdown',
+        value: 35,
+        fill: '#083fa1',
+        linkURL: 'https://en.wikipedia.org/wiki/Markdown',
+    },
+]
+
+const getValue = (datum: LanguageUsageDatum) => datum.value
+const getColor = (datum: LanguageUsageDatum) => datum.fill
+const getLink = (datum: LanguageUsageDatum) => datum.linkURL
+const getName = (datum: LanguageUsageDatum) => datum.name
+
+export const CategoricalPieChart = () => (
```

Comment:
Applicable to other stories too: 

```suggestion
import { Story } from '@storybook/react'

export const CategoricalPieChart: Story = () => (
```